pkg/proxy/ipvs/proxier.go

```diff
@@ -1357,31 +1366,43 @@ func (proxier *Proxier) syncProxyRules() {
 			}
 
 			// Build ipvs kernel routes for each node ip address
-			nodeIPs, err := proxier.ipGetter.NodeIPs()
+			nodeIPs := make([]net.IP, 0)
+			addresses, err := utilproxy.GetNodeAddresss(proxier.nodePortAddresses, proxier.networkInterface)
 			if err != nil {
-				glog.Errorf("Failed to get node IP, err: %v", err)
+				glog.Errorf("Failed to get node ip address matching nodeport cidr")
+				continue
+			}
+			addressList := addresses.UnsortedList()
+			if len(addressList) == 1 && utilproxy.IsZeroCIDR(addressList[0]) {
```

Comment:
I think these len=1 checks are all broken as soon as dual-stack lands.  Can we just not special-case this and let the 0 ranges behave like any other?